packages/ckeditor5-link/src/linkimageui.js

```diff
@@ -12,10 +12,16 @@ import Image from '@ckeditor/ckeditor5-image/src/image';
 import LinkUI from './linkui';
 import LinkEditing from './linkediting';
 
+import ButtonView from '@ckeditor/ckeditor5-ui/src/button/buttonview';
+
+import linkIcon from '../theme/icons/link.svg';
+
+const linkKeystroke = 'Ctrl+K';
```

Comment:
This one should be a global `LINK_KEYSTROKE` exported by `ckeditor5-link/src/utils` and shared between the plugins.